backend-lib/Cargo.lock

```diff
@@ -334,16 +409,16 @@ dependencies = [
 
 [[package]]
 name = "bip32"
-version = "0.5.2"
+version = "0.6.0-pre.1"
```

Comment:
See [here](https://github.com/zcash/librustzcash/pull/1666#discussion_r1932738755) for why we can't use 0.5.3.